interfaces/utils/path_patterns_test.go

```diff
@@ -92,12 +95,50 @@ func (s *pathPatternsSuite) TestRegexCreationUnhappy(c *C) {
 	for _, testData := range data {
 		pattern := testData.pattern
 		expectedError := testData.expectedError
-		pathPattern, err := utils.NewPathPattern(pattern)
+		const allowCommas = false
+		pathPattern, err := utils.NewPathPattern(pattern, allowCommas)
 		c.Assert(pathPattern, IsNil, Commentf("%s", pattern))
 		c.Assert(err, ErrorMatches, expectedError, Commentf("%s", pattern))
 	}
 }
 
+func (s *pathPatternsSuite) TestCommasInRegex(c *C) {
```

Comment:
Preference: I think it is nice to try and name Test[Function][Specific Focus]
e.g. something like: `TestCreateRegexWithCommas`

Reply:
Thanks for the suggestion!